mtags/src/main/scala/scala/meta/internal/metals/ScaladocIndexer.scala

```diff
@@ -125,6 +125,14 @@ class ScaladocIndexer(
 
 object ScaladocIndexer {
 
+  /**
+   * Extracts Scaladoc from Scala source code.
+   *
+   * @param fn callback function for calculated SymbolDocumentation
+   * @param occFn (optional) callback function for SymbolOccurrence
```

Comment:
I think after last changes you no longer have 'occFn'

Reply:
Thanks! Fixed :)